.studio/common

```diff
@@ -227,22 +227,6 @@ function rebuild() {
   return $err
 }
 
-document "build_all" <<DOC
```

Comment:
This function could not have been working for people since many of the packages it would have built do not build.  Thus, I've removed it since it implies people don't use it.